manifests/stage-nv-peer-mem-driver/0050_nv-peer-mem-driver-ds.yaml

```diff
@@ -43,6 +43,8 @@ spec:
         - key: nvidia.com/gpu
           operator: Exists
           effect: NoSchedule
+      serviceAccount: gpudirect-driver
```

Comment:
the rest of the project uses only `serviceAccountName`
can this be omitted as its deprecated ?

Reply:
Done